plugins/woocommerce/src/Internal/ProductAttributesLookup/DataRegenerator.php

```diff
@@ -71,6 +78,15 @@ final public function init( LookupDataStore $data_store ) {
 		$this->data_store = $data_store;
 	}
 
+	/**
+	 * Check if the last regeneration step failed.
+	 *
+	 * @return bool True if the last regeneration step failed.
+	 */
+	public function get_last_regeneration_step_feiled() {
```

Comment:
```suggestion
	public function get_last_regeneration_step_failed() {
```

Typo in method name (plus its usages elsewhere)